### ISSUE TITLE
where to find /cnn_data/yolov3_model/conv2d_53_w.npy used in graph_yolov3.cpp under examples ?

### ISSUE THREAD

**Issue by chetansarmak** [ARM-software/ComputeLibrary#741]
**Platform:Raspberry pi

**Operating System: Ubuntu mate

Where can I find the npy's for yolov3 referred in the graphyolov3.cpp code under examples folder?
How can I generate them from weight files (.weights to .npy )?

I have tried to convert the weights to .ckpt and used the _tensorflow_data_extractor.py_  to convert .ckpt's to .npy files .But, the converted files are different from the ones used in the examples/graph_yolov3.cpp code .
I couldn't find any proper documentation for using the  graph_yolov3.cpp
weights used : https://pjreddie.com/media/files/yolov3.weights

**Comment by drashmi29** [comment]
I am facing the same issue.
I have converted .cktpt to.npy. But I am not able to get _var.npy and _mean.npy files for batch normalization layers using the conversion script tensorflow_data_extractor.py.
Please suggest the solution!!!
